video_xblock/__init__.py

```diff
@@ -2,7 +2,7 @@
 Video xblock module.
 """
 
-__version__ = '0.10.1'
+__version__ = '0.11.1'
```

Comment:
Does our current version mean we do not have a stable v1.0.0 release? Maybe it is time?

Reply:
done